source/solution_deploy/source/wait_provider.py

```diff
@@ -0,0 +1,59 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Custom resource provider that waits for a specified time, then returns success"""
```

Comment:
Not in the custom resource provider. It is supposed to be general purpose.

Where would someone be looking to find that information?

Reply:
The use case is to wait in cloudformation deployment - this can have an effect on when resources are deployed with the addition of dependency links. We do some silly stuff to get ssm documents to deploy, but the details of that implementation aren't here. This is just a tool we use to inject waits.